functions/Get-DbaDbView.ps1

```diff
@@ -63,56 +66,49 @@ function Get-DbaDbView {
 
         Gets the views for the databases on Sql1 and Sql2/sqlexpress
 
+    .EXAMPLE
+        PS C:\> Get-DbaDatabase -SqlInstance Server1 -ExcludeSystem | Get-DbaDbView
+
+        Pipe the databases from Get-DbaDatabase into Get-DbaDbView
+
     #>
     [CmdletBinding()]
     param (
-        [parameter(Mandatory, ValueFromPipeline)]
```

Comment:
This line needs to stay in place.

Reply:
I removed this because with it I got this error.. does it need changing?
```
Get-DbaDbView : The input object cannot be bound because it did not contain the information required to bind all mandatory parameters:  SqlInstance
```

Reply:
Removing it will break folks ability to use do the following:

```
"server1","server2" | Get-DaDbView ....
```

The example we provide that does this will no longer be valid.